sde-cookbook/dlt-viewer/install_dlt-viewer.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) Luxoft 2018
+#
+# All rights reserved
```

Comment:
The PELUX-SDE is MPLv2, so this copyright notice should probably be changed to reflect this.

Reply:
@jeremiah already review those license that come from another project. I don't know if I have to change once become public and if I have to update even inside the other internal project, maybe can you explain me the right procedure?

Reply:
@erikboto What changes were you thinking of here? I don't see any issue with the copyright assignment line, i.e. line 3. I do see an issue with line 5 since all rights are not reserved, so I would remove line 5. I would add a line like ```SPDX-License-Identifier: MPL=2.0``` but this is not a super-strong requirement.